libibverbs/init.c

```diff
@@ -667,18 +667,27 @@ int ibverbs_init(void)
 {
 	char *env_value;
 
-	if (getenv("RDMAV_FORK_SAFE") || getenv("IBV_FORK_SAFE"))
+	bool want_ibv_fork_init = false;
+	env_value = getenv("RDMAV_FORK_SAFE");
+	want_ibv_fork_init = env_value && (strcmp(env_value, "0") != 0);
```

Comment:
Please just make a function to query an environment variable and return a bool if it is to be true or false. Then it is just

`
want_ibv_work_init = ibv_check_env("RDMAV_FORK_SAFE") || ibv_check_env("IBV_FORK_SAFE");
`

Reply:
Great suggestion!  Done.